### DISCUSSION TITLE
Hello all Can you help me return the money? I sent Ethereum to buy Manta but I have not received the coins for a month yet

### DISCUSSION THREAD

**Discussion by MOIUJIBKHALIL** [blockscout/blockscout#9433] [General]
<html><body>

  | 0xcfa120f8493ca438b32e7a3c5cd391e6e5682c587e38562e73b49d6391ce4f98 | Transfer | 19028044 | 33 days 22 hrs ago | 0x144A14...EaC863D1 | OUT | 0x95CeF1...1aC544E5 | 0.25031563 ETH |  
-- | -- | -- | -- | -- | -- | -- | -- | -- | --

</body>
</html>[0xcfa120f8493ca438b32e7a3c5cd391e6e5682c587e38562e73b49d6391ce4f98](https://etherscan.io/tx/0xcfa120f8493ca438b32e7a3c5cd391e6e5682c587e38562e73b49d6391ce4f98)
	Transfer	[19028044](https://etherscan.io/block/19028044)	33 days 22 hrs ago	
0x144A14...EaC863D1
	OUT	
[0x95CeF1...1aC544E5](https://etherscan.io/address/0x95cef13441be50d20ca4558cc0a27b601ac544e5)
	0.25031563 ETH

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
